props/saber_fett263_buttons.h

```diff
@@ -4336,6 +4368,26 @@ SaberFett263Buttons() : PropBase() {}
         }
         return true;
 
+      case EVENTID(BUTTON_POWER, EVENT_FOURTH_CLICK_LONG, MODE_ON):
+        if (menu_) return true;
+        if (fusor.angle1() < - M_PI / 3) {
```

Comment:
this could be written as:
ChangeStyleNumberAllBlades(fusor.angle1() < -M_PI / 3 ? -1 : 1);

Reply:
thanks, updated